### ISSUE TITLE
Ensure faction based Meltdown IDs match code

### ISSUE THREAD

**Issue by Maelstromeous** [ps2alerts/aggregator#138]
Currently there are a lot of questions in the air how the unstable metagame alerts trigger. We need to gather the data and figure out which triggers what.

We have the following list of Census Metagame IDs which we don't know what maps to what currently: 

```
    ESAMIR_UNSTABLE_MELTDOWN = 176,
    HOSSIN_UNSTABLE_MELTDOWN = 177,
    AMERISH_UNSTABLE_MELTDOWN = 178,
    INDAR_UNSTABLE_MELTDOWN = 179,

    ESAMIR_UNSTABLE_MELTDOWN_2 = 186,
    HOSSIN_UNSTABLE_MELTDOWN_2 = 187,
    AMERISH_UNSTABLE_MELTDOWN_2 = 188,
    INDAR_UNSTABLE_MELTDOWN_2 = 189,

    ESAMIR_UNSTABLE_MELTDOWN_3 = 190,
    HOSSIN_UNSTABLE_MELTDOWN_3 = 191,
    AMERISH_UNSTABLE_MELTDOWN_3 = 192,
    INDAR_UNSTABLE_MELTDOWN_3 = 193,
```

**Comment by leonhard-s** [comment]
Hey there - according to my notes, the first block should be Liberation (i.e. NC-triggered), the second block Enlightenment (VS-triggered), and the final one Superiority (i.e. TR-triggered).

I haven't done any testing on this recently, but I don't see why these would have changed since their implementation. I'll set up a small logger for these events to confirm once I de-bork my own websocket client, I'll report back if I find any errors.

Just in case, here's the full list I've cobbled together some months ago: https://github.com/leonhard-s/auraxium/wiki/Alert-ID-Reference

**Comment by Maelstromeous** [comment]
@leonhard-s awesome! Thanks for the info, I'll keep this ticket open as a reminder to me to update the comments within the code and any other code alterations I need to make.

**Comment by Maelstromeous** [comment]
Resolved via 4e6b031d709601f54e53acb5795e85a3ebf40221
